src/flake8/checker.py

```diff
@@ -443,14 +443,26 @@ def _extract_syntax_information(exception: Exception) -> Tuple[int, int]:
             token = ()
             row, column = (1, 0)
 
-        if column > 0 and token and isinstance(exception, SyntaxError):
+        if (
+            column > 0
+            and token
+            and isinstance(exception, SyntaxError)
+            and len(token) == 4  # Python 3.9 or earlier
```

Comment:
can this be `>=` 4 or does the `physical_line` bit not matter in 3.10?

Reply:
So `token[3]` will always be the string we care about. I left it as `[3]` because that's probably safer than `[-1]` and then I realized that the offsets changed and might be more accurate on 3.10 so we probably don't need to muck with them.